pkg/frontend/frontend.go

```diff
@@ -401,6 +405,12 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 
 	s.Methods(http.MethodPost).HandlerFunc(f.postAdminOpenShiftClusterDrainNode).Name("postAdminOpenShiftClusterDrainNode")
 
+	s = r.
```

Comment:
If we ever decide to expose this via the cluster manager, then we should remove the `/admin` route and use `api-version=admin` logic to route accordingly.